pkg/vsphere/vm/vm.go

```diff
@@ -262,10 +263,21 @@ func (vm *VirtualMachine) DeleteExceptDisks(ctx context.Context) (*object.Task,
 	}
 
 	disks := devices.SelectByType(&types.VirtualDisk{})
-
 	err = vm.RemoveDevice(ctx, true, disks...)
 	if err != nil {
-		return nil, err
+		switch f := err.(type) {
+		case task.Error:
+			switch f.Fault().(type) {
+			case *types.ConcurrentAccess:
+				log.Errorf("RemoveDevice failed for %s with ConcurrentAccess error. Ignoring it", vm)
```

Comment:
Just curious, does this mean someone else is removing device? Cause the device removing is critical for us, I have to confirm for this. 😄